apps/heureka/src/components/Services/Services.tsx

```diff
@@ -26,13 +27,16 @@ export type ServiceType = {
   serviceOwners: string[]
 }
 
-const defaultFilterSettings = {
-  selectedFilters: [],
-  searchTerm: "",
+const initializeFilterSettings = (initialFilters?: InitialFilters): FilterSettings => {
```

Comment:
because it is a function and functions perform actions so I'd suggest to start it's name with a verb like `getInitialFilters`.